admin/tools/docker/4_build-configured-images.sh

```diff
@@ -77,9 +80,13 @@ sed -i 's%{{COMMENT_ON_WORKER_OPT}}%%g' "$DOCKERFILE"
 TAG="${DOCKER_IMAGE}_master_${MASTER}"
 printf "Building master image %s from %s\n" "$TAG" "$DOCKERDIR"
 docker build --tag="$TAG" "$DOCKERDIR"
-docker push "$TAG"
+printf "Image %s built successfully\n" "$TAG"
+
+if [ "$PUSH_TO_HUB" = "true" ] ; then 
```

Comment:
The space before the `;` is a bit unusual.  There is trailing whitespace before the newline.